DigitalLearningSolutions.Web/Views/Frameworks/Developer/_SignpostingResourceCard.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@using DigitalLearningSolutions.Data.Models.LearningHubApiClient;
+@model ResourceMetadata
+
+@{
+    string DisplayText(string input)
+    {
+      return System.Text.RegularExpressions.Regex.Replace(input ?? String.Empty, "<.*?>", String.Empty);
+    };
+    string DisplayTagColour(string resourceType)
+    {
+      var colours = new string[] { "white", "grey", "green", "aqua-green", "blue", "purple", "red", "orange", "yellow" };
+      int hash = resourceType.ToCharArray().Sum(c => c) % colours.Length;
+      return colours[hash];
+    }
+}
+<details class="nhsuk-details nhsuk-expander  nhsuk-u-margin-bottom-3">
+  <summary class="nhsuk-details__summary">
+    <span class="nhsuk-details__summary-text">
+      @DisplayText(Model.Title)
+    </span>
+  </summary>
+  <div class="nhsuk-card__content nhsuk-u-padding-bottom-3 nhsuk-u-padding-top-0 nhsuk-u-margin-1">
+    <div class="nhsuk-grid-row nhsuk-u-margin-left-1 nhsuk-u-clear">
+      <strong class="nhsuk-tag nhsuk-tag--@(DisplayTagColour(Model.ResourceType))">@Model.ResourceType</strong>
+    </div>
+    <div class="nhsuk-u-margin-left-1 nhsuk-u-margin-top-3 nhsuk-u-margin-bottom-3">
+      @DisplayText(Model.Description)
```

Comment:
I think we have to swap this out for `@Html.Raw(Model.Description)`, there bits of HTML left in the text as it is. `&nbsp;` is the most common example but there are others too.
The descriptions that I've checked look okay as HTML but we could enclose them in an [NHS design system card](https://service-manual.nhs.uk/design-system/components/card) to improve the appearance.